development/values.yaml

```diff
@@ -13,12 +13,7 @@ git:
   ref: master
 
 cacheService: false
-cacheUseAuth: false
-configRepositoryId: false
-deployKey: false
-gitLabApiUrl: false
-gitLabPersonalAccessToken: false
-jwtVerifyPublicKeyLocation: false
-residenciesParentRepositoryId: false
-templateRepositoryId: false
-trustedClientKey: false
\ No newline at end of file
+ompGitlabApiUrl: https://omp-git-api-omp-dev.apps.s11.core.rht-labs.com
+jwtPublicKeyLocation: https://sso-omp-jasee.apps.s11.core.rht-labs.com/auth/realms/omp/protocol/openid-connect/certs
+jwtIssuer: https://sso-omp-jasee.apps.s11.core.rht-labs.com/auth/realms/omp
```

Comment:
On both this and the readme, I think Øystein wants us to leave dummy values in our public repos... We have example deployments with our real FQDNs in our GitLab wiki for reference

Reply:
good catch.  the dummy values are back in.